client/src/styles.js

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from '@material-ui/core/styles'
+import grey from '@material-ui/core/colors/grey'
+import green from '@material-ui/core/colors/green'
+
+const useStyles = makeStyles((theme) => ({
+    appbar: {
+        borderBottom: `1px solid ${grey['300']}`,
+        backgroundColor: '#fff'
```

Comment:
Here again,  I feel all colors should be standardized.

"theme.palette.primary.shadow" for the grey, to maintain a consistant color across all pages.
 and for consistancy, "theme.palette.common.black" and "theme.palette.common.white" can be used.